godot-cell/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "godot-cell"
+version = "0.1.0"
+edition = "2021"
+
+[dev-dependencies]
+proptest = "1.4.0"
```

Comment:
you can look here for last time i ran the full ci on my fork:
https://github.com/lilizoey/gdextension/actions/runs/6997929311
miri-test took 47 seconds there.

Reply:
29s of which were spent on dependencies and compilation 😛

No, 47s for that job is fine. But we need to keep in mind that `[dev-dependencies]` is not only used for miri, but also for unit tests (and examples or benchmarks, which are not applicable here). So this will have a ripple effect on all 3 unit-test jobs, at least when they're not cached.

Could we use something like [`[target.'cfg(...)'.dev-dependencies]`](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#development-dependencies)?

Reply:
honestly the easiest way to make sure this doesn't have this ripple effect is to just add a feature which enables proptest and a specific CI job that runs proptest. so i might just do that.